index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio | Edwin Gichuhi</title>
+    </head>
+
+    <body>
+        <header>
+            <nav class="toolbar">
+                <a href="#">John Doe</a>
+                <img src="./images/Icon-Menu.png" alt="menu">
+            </nav>
+        </header>
+
+        <section class="headline">
+            <h1>Hey There. I'm Alex</h1>
+            <h2>I’am a Software Developer</h2>
+            <h3>I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.</h3>
+
+            <ul class="socials">
+                <li>
+                    <a href="">
+                        <img src="./images/Icon-GitHub.png" alt="Github">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./images/Icon-Linkedin.png" alt="Github">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./images/Vector.png" alt="Github">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./images/Icon-Twitter.png" alt="Github">
+                    </a>
+                </li>
+                <li>
+                    <a href="">
+                        <img src="./images/Icon-Meta.png" alt="Github">
+                    </a>
+                </li>
+            </ul>
+
+            <img src="./images/after-socials.png" alt="after-socials-img" class="after-social-img">
```

Comment:
- [ ] Kindly make sure to  CSS `background-image` property to display your background image of this section. Kindly [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.